### PR TITLE
Appendix A Validation suggested editorial updates

### PR DESCRIPTION
**Pull request by grantnoble**
Wording changes, some grammar, and line length changes

***
<a href="https://pr-preview.s3.amazonaws.com/grantnoble/vc-data-model/pull/693.html" title="Last updated on Jul 16, 2019, 12:28 AM UTC (507a9cf)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/693/96dc09e...grantnoble:507a9cf.html" title="Last updated on Jul 16, 2019, 12:28 AM UTC (507a9cf)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4987,18 +4986,18 @@ <h3>Issuer</h3>
...
 
       <section class="informative">
         <h3>Issuance Date</h3>
 
         <p>
-The <code>issuanceDate</code> is expected to be within an expected range for
-the <a>verifier</a>. For example, a <a>verifier</a> can ensure that the
-issuance date of a <a>verifiable credential</a> is not in the future.
+The <code>issuanceDate</code> is expected to be within an expected range for the
+<a>verifier</a>. For example, a <a>verifier</a> can ensure the issuance date of
```
Comment (TallTed) [inline index.html]: `ensure the issuance date` -> `check that the issuance date`

File: index.html [hunk 2]
```diff
@@ -5007,11 +5006,11 @@ <h3>Proofs (Signatures)</h3>
...
 The cryptographic mechanism used to prove that the information in a
-<a>verifiable credential</a> or a <a>verifiable presentation</a> has not been
-tampered with is called a <em>proof</em>. There are many types of
-cryptographic proofs including, but not limited to, digital signatures,
-zero-knowledge proofs, Proofs of Work, and Proofs of Stake. In general, when
-verifying proofs implementations are expected to ensure that:
+<a>verifiable credential</a> or <a>verifiable presentation</a> was not
+tampered with is called a <em>proof</em>. There are many types of cryptographic
+proofs including, but not limited to, digital signatures, zero-knowledge proofs,
+Proofs of Work, and Proofs of Stake. In general, when verifying proofs
```
Comment (TallTed) [inline index.html]: `when verifying proofs` -> `when verifying proofs,`

File: index.html [hunk 3]
```diff
@@ -5073,7 +5071,7 @@ <h3>Expiration</h3>
 
         <p>
 The <code>expirationDate</code> is expected to be within an expected range
-for the <a>verifier</a>. For example, a <a>verifier</a> can ensure that the
+for the <a>verifier</a>. For example, a <a>verifier</a> can ensure the
```
Comment (TallTed) [inline index.html]: `can ensure the ` -> `can check that the`

File: index.html [hunk 4]
```diff
@@ -5083,29 +5081,28 @@ <h3>Status</h3>
...
         <h3>Fitness for Purpose</h3>
 
         <p>
-The custom <a>properties</a> in the <a>verifiable credential</a> are appropriate
-for the <a>verifier's</a> purpose. For example if a <a>verifier</a> needs to
-determine whether a <a>subject</a> is older than 21 years of age, they might
-rely on a specific <code>birthdate</code> <a>property</a>, or more abstract
-<a>properties</a>, such as <code>ageOver</code>.
+Whether the custom <a>properties</a> in the <a>verifiable credential</a> are
+appropriate for the <a>verifier's</a> purpose. For example if a <a>verifier</a>
```
Comment (TallTed) [inline index.html]: `For example if` -> `For example, if`

File: index.html [hunk 5]
```diff
@@ -5083,29 +5081,28 @@ <h3>Status</h3>
...
+needs to determine whether a <a>subject</a> is older than 21 years of age, they
+might rely on a specific <code>birthdate</code> <a>property</a>, or more
```
Comment (TallTed) [inline index.html]: `<a>property</a>, or` -> `<a>property</a> or on`

File: index.html (hunk 1 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 2 above)
Comment (grantnoble) [reply index.html]: A comma is not specifically required here, but ok.

File: index.html (hunk 3 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 4 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 5 above)
Comment (grantnoble) [reply index.html]: Done.

**Comment by grantnoble** [comment]
Updates completed.

File: index.html [hunk 6]
```diff
@@ -5083,29 +5081,28 @@ <h3>Status</h3>
...
       <section class="informative">
         <h3>Fitness for Purpose</h3>
 
         <p>
-The custom <a>properties</a> in the <a>verifiable credential</a> are appropriate
-for the <a>verifier's</a> purpose. For example if a <a>verifier</a> needs to
-determine whether a <a>subject</a> is older than 21 years of age, they might
-rely on a specific <code>birthdate</code> <a>property</a>, or more abstract
-<a>properties</a>, such as <code>ageOver</code>.
+Whether the custom <a>properties</a> in the <a>verifiable credential</a> are
```
Comment (brentzundel) [inline index.html]: This does not seem to be a complete sentence.
```
Whether the custom <a>properties</a> in the <a>verifiable credential</a> are
appropriate for the <a>verifier's</a> purpose.
```

Comment (grantnoble) [reply index.html]: How about "Fitness for purpose is about whether the custom properties... "?

Comment (brentzundel) [reply index.html]: Works for me

Comment (grantnoble) [reply index.html]: Updated.

**Pull request merged**